service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/MapperService.java

```diff
@@ -0,0 +1,54 @@
+package uk.nhs.adaptors.gp2gp.ehr.mapper;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.Observation;
+import org.hl7.fhir.dstu3.model.Resource;
+import org.hl7.fhir.dstu3.model.ResourceType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import uk.nhs.adaptors.gp2gp.common.service.RandomIdGeneratorService;
+
+@Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+@Component
+public class MapperService {
```

Comment:
I don't understand the purpose of this service. Mapping all narrative statements together is not a GP2GP requirements.

Reply:
I need to somewhere initialise message context and clear it after transforming